.github/workflows/notebook-pr.yaml

```diff
@@ -38,6 +38,7 @@ jobs:
           python -m pip install --upgrade pip wheel
           pip install -r requirements.txt
           pip install nbconvert pillow flake8 fuzzywuzzy[speedup]
+          pip install pyyaml
```

Comment:
for reference, you could just add pyyaml to the end of the line above, but this is fine